sections/all_map_locations.tex

```diff
@@ -69,16 +72,18 @@ \subsection*{Symbols on the Map}
 
   \phantom{j}\textbf{Treasure}\\
   \framedimage[\linewidth]{\map_locations/treasure_symbol.jpg}\\
+}
+\end{multicols*}
 
-  \bigskip
+\pagebreak
 
-  \phantom{j}\textbf{Fountain of Youth}\\
-  \framedimage[\linewidth]{\map_locations/fountain_of_youth.jpg}\\
+The effects of the following \textbf{Visitable} fields are explained by the symbols on the previous page:
```

Comment:
```suggestion
```
BTW, is pagebreak above it necessary?

Reply:
This sentence stays, sorry. I added it intentionally.

It's a new page, and it just has a list of random images with zero explanation. This list of images is even more confusing as is than in the official book.

> is pagebreak above it necessary?

There's just not enough space on the previous page

Reply:
I'm really not a fan of overexplaining things like that, because two people found it hard to read.

It's not about room, all the content should move gracefully to the next page without `\pagebrak`.

Reply:
> It's not about room, all the content should move gracefully to the next page without \pagebrak.

Generally I would agree for text which is read from top to bottom. But this is a random list of images without any description. Usage of the list is to find correct image matching the field to understand what it does. And here it does... nothing?